apps/home/templates/apps_home/blocks/teaser_block_two_columns.html

```diff
@@ -5,7 +5,7 @@
 <div class="modul-servicepanel fullwidth ds-block {% if self.background_color %}ds-block--{{ self.background_color }}{% else %}ds-block__body-bg{% endif %}">
     <div class="servicepanel__main flexgrid grid--2 text-center">
         <div class="">
```

Comment:
empty class atrribute, should be removed?

Reply:
this block will likely be deleted when we get final go ahead on designs so I thought to just leave it for now